src/js/components/common/buttons/PrimaryButton.js

```diff
@@ -0,0 +1,33 @@
+/* @flow */
+import {darken} from "polished"
+import styled from "styled-components"
+
+import type {Styled} from "../../../types/styled"
+
+const bg = "#4b91e2"
```

Comment:
Does this "magic color" correspond to anything we use anywhere else?

Reply:
Oops, yeah I need to fix that.